content/docs/main/docs/antrea-ipam.md

```diff
@@ -88,8 +88,14 @@ IPPool annotation, or when the `AntreaIPAM` feature is disabled.
 
 To enable flexible IPAM, you need to enable the `AntreaIPAM` feature gate for
 both `antrea-controller` and `antrea-agent`, and set the `enableBridgingMode`
-configuration parameter of `antrea-agent` to `true`. The needed changes in the
-Antrea deployment YAML are:
```

Comment:
"the Antrea deployment YAML" here means the manifest YAML used to deploy Antrea, not the antrea-controller deployment resource. But it's good to make it more specific.